2018-08-31 10:06:48,924 - TC_login.py - testLogin01 > 29 > INFO：第一条用例：2018-08-31 10:06:382018-08-31 10:06:48successful
2018-08-31 10:10:24,093 - TC_login.py - testLogin01 > 32 > ERROR：第一条测试用例失败
2018-08-31 10:48:56,034 - TC_login.py - testLogin01 > 30 > INFO：第一条用例：2018-08-31 10:48:432018-08-31 10:48:560:00:13successful
2018-08-31 10:51:00,683 - TC_login.py - testLogin01 > 30 > INFO：第一条用例：2018-08-31 10:50:452018-08-31 10:51:000:00:15successful
2018-08-31 10:51:00,774 - TC_login.py - testLogin01 > 30 > INFO：第一条用例：2018-08-31 10:50:462018-08-31 10:51:000:00:14successful
2018-08-31 10:52:43,711 - TC_login.py - testLogin01 > 30 > INFO：第一条用例：2018-08-31 10:52:302018-08-31 10:52:430:00:13successful
2018-08-31 11:00:44,872 - TC_login.py - testLogin01 > 32 > INFO：第一条用例：开始时间：2018-08-31 11:00:32结束时间：2018-08-31 11:00:44结束时间：0:00:12测试结果：successful
